### PR TITLE
Documentation for serializing resources without render

### PR DESCRIPTION
**Pull request by PericlesTheo**
https://github.com/rails-api/active_model_serializers/issues/1101

### CODE REVIEW

**Comment by NullVoxPopuli** [comment]
I like this. good work!

File: README.md [hunk 1]
```diff
@@ -138,6 +138,29 @@ render json: @post, meta: { total: 10 }, meta_key: "custom_meta"
 
 `meta` will only be included in your response if you are using an Adapter that supports `root`, as JsonAPI and Json adapters, the default adapter (FlattenJson) doesn't have `root`.
 
+### Using a serializer without `render`
+
+At times, you might want to use a serializer without rendering it to the view. For those cases, you can create an instance of `ActiveModel::SerializableResource` with
+the resource you want to be serialized and call `.serializable_hash`.
+
+```ruby
+def create
```
Comment (NullVoxPopuli) [inline README.md]: for the sake of readme length, do you think this should be simplified?

having context is nice though.

@joaomdmoura, what are your thoughts?

Comment (PericlesTheo) [inline README.md]: Yea initially I had it a bit shorter

Comment (PericlesTheo) [inline README.md]: @NullVoxPopuli do you mean the code example or the text? Or both? :)

Comment (NullVoxPopuli) [inline README.md]: just the code example. the text is great! :)

Comment (PericlesTheo) [inline README.md]: How about? Less verbose and still has some contextual meaning
```ruby
def create
  @message = current_user.messages.create!(message_params)
   MessageCreationWorker.perform(serialized_message)
   head 204
end

def serialized_message
  ActiveModel::SerializableResource.new(@message).serializable_hash
end
```

Comment (NullVoxPopuli) [inline README.md]: that looks great!

Comment (PericlesTheo) [inline README.md]: :boom:

**Comment by NullVoxPopuli** [comment]
looks good!

**Comment by PericlesTheo** [comment]
@NullVoxPopuli cheers that was superfast! :heart:

File: README.md [hunk 2]
```diff
@@ -138,6 +138,23 @@ render json: @post, meta: { total: 10 }, meta_key: "custom_meta"
...
+  @message = current_user.messages.create!(message_params)
+   MessageCreationWorker.perform(serialized_message)
```
Comment (beauby) [inline README.md]: Indentation

**Comment by joaomdmoura** [comment]
:+1:

**Comment by Awatatah** [comment]
I am trying this and I am getting `NameError: uninitialized constant ActiveModel::SerializableResource`

**Comment by bf4** [comment]
@Awatatah That's because you presumably have a typo `ActiveModel::SerializableResource` should be `ActiveModelSerializers::SerializableResource`.  Please don't comment on years old issue. Please open a new issue if you continue to have trouble resolving this.

**Pull request merged**